rest/available-phone-numbers/local-basic-example-8/local-get-basic-example-8.5.x.cs

```diff
@@ -0,0 +1,28 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Rest.Api.V2010.Account.AvailablePhoneNumberCountry;
+using System.Linq;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var localAvailableNumber = LocalResource.Read("US",
```

Comment:
It's a typo, I meant to write `localAvailableNumbers`. I think the `Available` adds meaning as the name of the class does not specify it.